Backend/utils/MentiiLogging.py

```diff
@@ -0,0 +1,30 @@
+import os
+import time
+import logging
+from logging.handlers import TimedRotatingFileHandler
+
+PATH = '../logs/metniilog.log'
```

Comment:
AWS might have an issue with the relative path. We will probably need an absolute path. Maybe have a log path in the config file that is read and we write the log to there.